doc/index.rst

```diff
@@ -38,7 +38,7 @@ PypeIt |stars| |watch|
 
 |pypi| |DOI_latest| |arxiv| |astropy|
 
-**Version**: 1.3.1dev
+**Version**: |version|
```

Comment:
Sorry if I missed it, but where is this defined?

Reply:
that gets defined in `conf.py` and `sphinx` gets it from there. i just followed what the comment in `conf.py` said and it worked.

Reply:
Nice.